panther_analysis_tool/schemas.py

```diff
@@ -29,9 +29,10 @@
     r"\.Instance|Redshift\.Cluster|S3\.Bucket|WAF\.Regional\.WebACL|WAF\.WebACL)$"
 )
 LOG_TYPE_REGEX = Regex(
-    r"^(Apache\.AccessCombined|Apache\.AccessCommon|AWS\.ALB|AWS\.AuroraMySQLAudit|AWS"
```

Comment:
Maybe we want to bump or ignore line length in this stanza? This is kinda painful to shuffle when trying to maintain alphabetical order

Reply:
agreed, this is super annoying and hard to maintain